src/IronPythonExtension/IronPythonExtension.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.IO;
+using System.Linq;
+using System.Reflection;
+using Dynamo.Extensions;
+
+namespace IronPythonExtension
+{
+    public class IronPythonExtension : IExtension
+    {
+        private const string PythonEvaluatorAssembly = "DSIronPython";
+
+        public void Dispose()
+        {
+        }
+
+        public string UniqueId => "D7B449D7-4D54-47EF-B742-30C7BEDFBE92";
+
+        public string Name => "IronPythonExtension";
+
+        public void Startup(StartupParams sp)
```

Comment:
SQ may complain about a code smell here saying that `sp` doesn't match with `startupParams` parameter name as defined in the interface.

Reply:
It actually matches because IExtension have it as `sp`